docs/topics/jvm/jvm-test-using-junit.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Test code using JUnit in JVM – tutorial)
+
+This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.
+
+The example has the [kotlin.test](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.
+
+To get started, first download and install the latest version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
+
+## Add dependencies
+
+1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](jvm-get-started.md#create-an-application).
+
+   > Specify **JUnit 5** as your test framework when creating a project.
+   >
+   {type="note"} 
+
+2. Open the `build.gradle(.kts)` file and add the following dependencies to the Gradle configuration to work with `kotlin.test` and `JUnit`:
+
+   <tabs>
+
+   ```groovy
+   dependencies {
+       // Other dependencies.
+       testImplementation 'org.jetbrains.kotlin:kotlin-test-junit5'
+       testImplementation 'org.junit.jupiter:junit-jupiter-api:5.6.0'
+       testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.6.0'
+   }
+   ```
+
+   ```kotlin
+   dependencies {
+       // Other dependencies.
+       testImplementation(kotlin("test-junit5"))
+       testImplementation("org.junit.jupiter:junit-jupiter-api:5.6.0")
+       testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.6.0")
+   }
+   ```
+
+   </tabs>
+
+3. Ensure, that **Project Wizard** automatically created the `test` task in the `build.gradle(.kts)` file:
+
+   <tabs>
+
+   ```groovy
+   test {
+       useJUnitPlatform()
+   }
+   ```
+
+   ```kotlin
+   tasks.test {
+       useJUnit()
```

Comment:
Also `...JUnitPlatform`

Reply:
Thanks!
I've updated the Groovy to JUnit 5 but forgot to update the Kotlin code block